samples/gui/lvgl/src/main.c

```diff
@@ -37,6 +37,32 @@ void main(void)
 	count_label = lv_label_create(lv_scr_act(), NULL);
 	lv_obj_align(count_label, NULL, LV_ALIGN_IN_BOTTOM_MID, 0, 0);
 
+	/*Create styles for the keyboard*/
```

Comment:
In case it is of any use I have a [sample](https://github.com/vanwinkeljan/zephyr/tree/sdl_gpio/samples/gui/lvgl_keypad) floating around that uses GPIO pins to control some switches maybe this could be extended to allow controle through the touch screen.